addons/l10n_fr/models/account_move.py

```diff
@@ -0,0 +1,17 @@
+from odoo import models, api
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.depends("country_code", "move_type")
+    def _compute_show_delivery_date(self):
+        super()._compute_show_delivery_date()
+
```

Comment:
You can add the # EXTENDS 'account' comment so that we can directly know from where the compute is extended :smile:, also the blank space is not really needed but no big deal